src/ebay-infotip/ebay-infotip.tsx

```diff
@@ -67,27 +48,35 @@ const EbayInfotip: FC<InfotipProps> = ({
         throw new Error(`EbayInfotip: Please use a EbayInfotipContent that defines the content of the infotip`)
     }
 
-    const { children: contentChildren, ...contentProps } = content.props
+    const { children: contentChildren, ...contentProps } = elementProps(content)
+
+    const buttonProps = {
+        ref: buttonRef,
+        onClick: toggleTooltip,
+        disabled,
+        variant,
+        'aria-label': ariaLabel,
+        'aria-expanded': isExpanded,
+        icon
```

Comment:
Should we also install eslint for typescript, so  it can also give us error when we forget `;`

Reply:
@ronikadriu we don't use `;` in this project historically :)